ci-operator/step-registry/openshift-extended/test/openshift-extended-test-commands.sh

```diff
@@ -320,6 +320,7 @@ function run {
     # summarize test results
     echo "Summarizing test results..."
     failures=0 errors=0 skipped=0 tests=0
+    [[ -e "${ARTIFACT_DIR}" ]] || mkdir --parents "${ARTIFACT_DIR}"
```

Comment:
@liangxia I take this step as example to raise my concern (same concern for other step.):
when step run here, if there is no artifacts directory, it means no case is executed.
if we create this directory here, the follow code does not create junit xml. and then trap function warn_0_case_executed also does not create junit xml because this directory is already existing (created with this line.), so finally still no junit xml.

so,  `[[ -e "${ARTIFACT_DIR}" ]] || mkdir --parents "${ARTIFACT_DIR}"` to `[[ -e "${ARTIFACT_DIR}" ]] || exit 0` to skip to grep this directory and finally run into trap function to generate junit xml.

Reply:
warn_0_case_executed checks whether there are files in `${ARTIFACT_DIR}`, so we do not care where is the folder created.

Reply:
Rethink the issue, I agree that `exit 0` is much better.